### DISCUSSION TITLE
Customtkinter not working in both VS Code and Pycharm

### DISCUSSION THREAD

**Discussion by abhidhakal** [TomSchimansky/CustomTkinter#1640] [General]
It says "No module named customtkinter" even though it is installed.

Traceback (most recent call last):
  File "/Users/jnjndkds/Desktop/python/project.py", line 2, in <module>
    import customtkinter
ModuleNotFoundError: No module named 'customtkinter'

**Comment by abhaymakes** [comment]
Have you tried uninstalling and reinstalling the module?

**Reply by abhidhakal** [reply]
i tried it. didnt work

**Comment by TechPro424** [comment]
Same issue for me
Works when I run it using py command in command line though

**Comment by TechPro424** [comment]
Fixed in VSC by using a virtual environment or venv by following this site https://code.visualstudio.com/docs/python/environments
and selecting python 3.11.5 as interpreter and then installing customtkinter using pip install
